### PR TITLE
feat(FX-4248): add Activity screen

### PR DESCRIPTION
**Pull request by dimatretyak**
### Demo
https://user-images.githubusercontent.com/3513494/191985250-347a8018-1342-4215-86e6-4fd20e2c7fa6.mp4

This PR resolves [FX-4248]

#### Acceptance Criteria

-

#### Setup Instructions

-

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [x] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- add Activities screen - dimatretyak

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[FX-4248]: https://artsyproduct.atlassian.net/browse/FX-4248?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/AppRegistry.tsx [hunk 1]
```diff
@@ -314,6 +315,7 @@ export type AppModule = keyof typeof modules
 
 export const modules = defineModules({
   Storybook: reactModule(StorybookUIRoot),
+  Activities: reactModule(Activities),
```
Comment (dimatretyak) [inline src/app/AppRegistry.tsx:318]: it seems to me that `Activities` is better suited as a name for the screen. what do you think about this?

Comment (dblandin) [reply src/app/AppRegistry.tsx:318]: **suggestion**: Maybe `Activity`?

Comment (dimatretyak) [reply src/app/AppRegistry.tsx]: Renamed

**Review by gkartalis — DISMISSED** [review]
Looks great! 

One request: can you also add though the route to `src/main/AndroidManifest.xml` like the docs [reference](https://github.com/artsy/eigen/blob/main/docs/adding_a_new_route.md#how-to-add-a-global-app-route)?

 The reason is for the deep linking to work if we enable them in the feature (sometimes we forget to add it to manifest and it takes some time to realize that this is the reason that they don't work)

**Review by dblandin — COMMENTED** [review]
Looks great :+1:

**Review by dblandin — APPROVED** [review]

**Comment by dimatretyak** [comment]
> One request: can you also add though the route to `src/main/AndroidManifest.xml` like the docs [reference](https://github.com/artsy/eigen/blob/main/docs/adding_a_new_route.md#how-to-add-a-global-app-route)?
> 
> The reason is for the deep linking to work if we enable them in the feature (sometimes we forget to add it to manifest and it takes some time to realize that this is the reason that they don't work)

oh, really good catch 🙏

**Comment by dimatretyak** [comment]
> The reason is for the deep linking to work if we enable them in the feature (sometimes we forget to add it to manifest and it takes some time to realize that this is the reason that they don't work)

addressed in [f21d5b8](https://github.com/artsy/eigen/pull/7394/commits/f21d5b8a59b7ef793d9ae56dbf0284d9e917b9d6)

**Pull request merged**